docs/root/configuration/http/http_filters/fault_filter.rst

```diff
@@ -36,6 +36,9 @@ The fault filter has the capability to allow fault configuration to be specified
 This is useful in certain scenarios in which it is desired to allow the client to specify its own
 fault configuration. The currently supported header controls are:
 
+* Request abort configuration via the *x-envoy-fault-abort-request* header. The header value
+  should be an integer that specifies the HTTP status code to return in response to a request
+  and must be in the range [200, 600).
```

Comment:
Worth pointing out that `header_abort` needs to be set (and including an RST link back to it).